README.md

```diff
@@ -269,7 +269,7 @@ interface on port `8080` below):
 
         RewriteEngine On
         # Websocket connections from the clients.
-        RewriteRule ^/standalone-signaling/spreed$ - [L]
+        RewriteRule ^/standalone-signaling/spreed/$ - [L]
```

Comment:
Thinking about this, now the rule is no longer applied to client requests (which are loading `/standalone-signaling/spreed` (without a trailing slash). Does it still work if you remove the line (and the comment above) completely?

Reply:
Without that line I Nextcloud does not complain and I can join calls, but the video connection is not working and chat messages are not loaded or delivered.

Reply:
Others seemed to have a similar problem with the nginx config as well: https://help.nextcloud.com/t/signaling-server-recommendations/52798/58?u=freed

Reply:
My [post](https://help.nextcloud.com/t/error-400-when-trying-to-access-high-performance-backend-signaling/106999/6)in the forum. I can only recommend the commit.